setup.py

```diff
@@ -139,7 +139,8 @@ def initialize_options(self) -> None:
 airflow_dependencies = [
     # Minimum version here is needed to avoid syntax error in setup.py
     # in 1.10.0
-    'apache-airflow>=1.10.1,<2'
+    'apache-airflow>=2',
```

Comment:
This file, one of the 3 largest in the project, increased in size to 369 lines.  The total size of those files is now 3411 lines (target: 1138).  Is this file complex enough to refactor?